server.js

```diff
@@ -14,6 +15,7 @@ const MONGODB_URI = process.env.MONGODB_URI || "mongodb://localhost/chefd";
 mongoose.connect(MONGODB_URI, { useNewUrlParser: true, useUnifiedTopology: true });
 
 require("./routes/apiRoutes")(app);
+app.use(require("./routes/authRoutes"));
```

Comment:
@ngiridhar1 Hey I'm not familiar with this syntax for routing, is this going to override our apiRoutes or will they both be used? Have you tested that both still work?